core/src/main/resources/org/apache/spark/ui/static/utils.js

```diff
@@ -272,3 +272,16 @@ function errorMessageCell(errorMessage) {
   const details = detailsUINode(isMultiline, errorMessage);
   return summary + details;
 }
+
+function stringAbbreviate(content, limit) {
+  if (content && content.length > limit) {
+    const summary = content.substring(0, limit) + '...';
+    // TODO: Reused stacktrace-details* style for convenience, but it's not really a stacktrace
```

Comment:
Please use IDed TODO with JIRA ID next time~

Reply:
Then, could you avoid to use `TODO`? TODO is a recognized keyword in the IDE.

Reply:
OK, I got it. Thank you for the explanation, @dongjoon-hyun